Detectors/MUON/MCH/Workflow/src/DataDecoderSpec.cxx

```diff
@@ -149,12 +149,25 @@ class DataDecoderTask
       auto flags = o2::raw::RDHUtils::getCRUID(rdhAny) & 0xFF00;
       auto endpoint = o2::raw::RDHUtils::getEndPointID(rdhAny);
       auto feeId = cruId * 2 + endpoint + flags;
+      auto linkId = o2::raw::RDHUtils::getLinkID(rdhAny);
       o2::raw::RDHUtils::setFEEID(rdhAny, feeId);
       orbit = o2::raw::RDHUtils::getHeartBeatOrbit(rdhAny);
       if (mPrint) {
         std::cout << mNrdhs << "--\n";
         o2::raw::RDHUtils::printRDH(rdhAny);
       }
+
+      uint64_t orbitInfo = orbit;
```

Comment:
@aphecetche I have created a separate `OrbitInfo` class with getter functions for orbit/linkID/feeID fields, and moved the creation out of the patch page function.

Concerning the use of `std::set`, the problem is that it does not provide a `data()` function to access the underlying storage buffer, so one would need to copy elements one-by-one instead of using a `memcpy()`. I have therefore kept the original `std::vector`.